addons/forum-post-snip/addon.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Nested forum post collapser",
+  "description": "Collapses forum quotes to a limit.",
```

The purpose of this addon can be confusing initially, so we need to be careful about how we word this. I'd suggest something like:
```suggestion
  "description": "When there are multiple forum quotes nested inside of each other, only shows a certain amount of them.",
```